renovate.json

```diff
@@ -1,5 +1,3 @@
 {
-  "extends": [
-    "config:base"
-  ]
+  "extends": ["config:base"]
```

Comment:
This change is due to prettier now being run on all files that it supports, including JSON files and markdown files